hawc/apps/animal/forms.py

```diff
@@ -758,9 +758,9 @@ class EndpointFilterForm(forms.Form):
     )
 
     sex = forms.MultipleChoiceField(
-        choices=models.AnimalGroup.SEX_CHOICES,
+        choices=constants.Sex.choices,
         widget=forms.CheckboxSelectMultiple,
-        initial=[c[0] for c in models.AnimalGroup.SEX_CHOICES],
+        initial=[c[0] for c in constants.Sex.choices],
```

Comment:
use ` ExperimentType.__members__.keys()` instead?

Reply:
Even better, it looks like I can just do `.values`!
![image](https://user-images.githubusercontent.com/46504563/144462914-8725e422-6bb9-4f00-a935-8e1794888826.png)